docs/docs/20-usage/25-workflows.md

```diff
@@ -124,19 +124,7 @@ depends_on:
 +runs_on: [ success, failure ]
 ```
 
-Some workflows don't need the source code, set the `skip_clone` tag to skip cloning:
-
-```diff
-
-pipeline:
-  notify:
-    image: debian:stable-slim
-    commands:
-      - echo notifying
-
-depends_on:
-  - deploy
-
-runs_on: [ success, failure ]
-+skip_clone: true
-```
+:::info
+Some workflows don't need the source code, like creating a notification on failure.
+Read more about `skip_clone` at [pipeline syntax](./20-pipeline-syntax.md#skip_clone)
+:::
```

Comment:
Do we even need this anymore after adding the `skip_clone` section? Maybe we can simply move the example to that section as well.

Reply:
I'd like the reminder here as most wont think of that while creating notification workflows i would say

Reply:
Never mind, missed that its a completely different file